core-bundle/contao/dca/tl_content.php

```diff
@@ -324,6 +325,13 @@
 			),
 			'sql'                     => "blob NULL"
 		),
+		'descriptionlistitems' => array
```

Comment:
Or - if this is going to be a key/value storage after all - what about reusing `data` (the one the `template` content element is using)?

Reply:
the one for lists is called `listitems`, thats just where I got this from. My first idea was to call it `dlitems` but not sure that's more descriptive …

Reply:
I'd just re-use `tl_content.data`. The key should be `dt` and the value `dd`.

Reply:
I honestly don't think this would fit the use case many people have. I'd assume if you need several terms you would want to style them individually as well and then you need another structure again. For the details, I would rather allow a RTE field so that you could have multiple paragraphs to structure things.

Reply:
>> I'd just re-use `tl_content.data`. The key should be `dt` and the value `dd`.
>
> as @ausi already pointet out, a description list can contain multiple `<dd>` per `<dt>`

This should be no problem by using the `data` column and a template similar to the one I mentioned in https://github.com/contao/contao/pull/5424#discussion_r1006192829